src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -274,7 +300,20 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithNonTightHtml()) {
+                    processTree(javadocTree);
+                }
+                else {
+                    final DetailNode nonTightHtmlTag = JavadocUtils.getNonTightHtmlTag(javadocTree);
+                    if (nonTightHtmlTag == null) {
+                        processTree(javadocTree);
+                    }
+                    else if (doesCheckViolateJavadocWithNonTightHtml()) {
+                        log(nonTightHtmlTag.getLineNumber(), nonTightHtmlTag.getColumnNumber(),
+                                "javadoc.unclosedHtml", nonTightHtmlTag.getText());
```

Comment:
we will do special Check for this later on. We can make separate issue on this.
Right now we do fix in "checkstyle framework" to help all Checks to execute on expected/stable AST.